client/store/cart.js

```diff
@@ -45,17 +41,32 @@ export const fetchItems = () => {
   return async dispatch => {
     try {
       const {data} = await Axios.get('/api/cart')
-      dispatch(getItems(data))
+      // console.log(data)
+      // console.log({ ...data.products[0]})
+      // console.log({...data.cartItems[0]})
+      let info = {}
+      for (let i = 0; i < data.products.length; i++) {
+        info['cartItem' + (i + 1)] = {
+          //Only grab needed info for each cart item
+          id: i + 1,
+          quantity: data.cartItems[i].quantity,
+          productId: data.cartItems[i].productId,
+          name: data.products[i].name,
+          image: data.products[i].image,
+          price: data.products[i].price
+        }
+      }
+      console.log(info)
```

Comment:
We'll need to remove console.logs and commented out console.logs, but that shouldn't stop this from being approved.